### PR TITLE
docs: clarify how we handle disabled plugin which is matched

### PR DESCRIPTION
**Pull request by spacewander**
### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [ ] Is this PR backward compatible?

### CODE REVIEW

**Review by nic-chen — APPROVED** [review]

File: doc/plugins.md [hunk 1]
```diff
@@ -21,14 +21,17 @@
...
+APISIX plugins are hot-loaded. No matter you add, delete or modify plugins, you don't need to restart the service.
 
 If your APISIX node has the Admin API turned on, just send an HTTP request through admin API:
 
 ```shell
 curl http://127.0.0.1:9080/apisix/admin/plugins/reload -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT
 ```
 
+Note: if you disable a plugin which has been configured as part of your rule (in the `plugins` field of `route`, etc.),
+the execution of it will be skipped.
```
Comment (juzhiyuan) [inline doc/plugins.md:33]:
```suggestion
its execution will be skipped
```

File: doc/zh-cn/plugins.md [hunk 2]
```diff
@@ -29,6 +29,9 @@ APISIX 的插件是热加载的，不管你是新增、删除还是修改插件
 curl http://127.0.0.1:9080/apisix/admin/plugins/reload -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT
 ```
 
+注意：如果你之前在路由规则里配置了某个插件（比如在 `route` 的 `plugins` 字段里面添加了它），然后
```
Comment (juzhiyuan) [inline doc/zh-cn/plugins.md:32]:
```suggestion
已经
```

**Review by juzhiyuan — APPROVED** [review]
LGTM

File: doc/zh-cn/plugins.md (hunk 2 above)
Comment (juzhiyuan) [reply doc/zh-cn/plugins.md:32]: `注意：如果你已经在路由规则里配置了某个插件（比如在 `route` 的 `plugins` 字段里面添加了它），然后`

File: doc/plugins.md [hunk 3]
```diff
@@ -21,14 +21,17 @@
...
+APISIX plugins are hot-loaded. No matter you add, delete or modify plugins, you don't need to restart the service.
 
 If your APISIX node has the Admin API turned on, just send an HTTP request through admin API:
 
 ```shell
 curl http://127.0.0.1:9080/apisix/admin/plugins/reload -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT
 ```
 
+Note: if you disable a plugin which has been configured as part of your rule (in the `plugins` field of `route`, etc.),
+the its execution will be skipped.
```
Comment (juzhiyuan) [inline doc/plugins.md:33]: Please remove `the`

Comment (juzhiyuan) [reply doc/plugins.md:33]: Or use then instead.

**Pull request merged**